packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -71,9 +71,9 @@ export default class QuanticGeneratedAnswer extends LightningElement {
    * A list of fields to fetch with the citations used to generate the answer.
    * @api
    * @type {string}
-   * @defaultValue `'sfid,sfkbid,sfkavid'`
+   * @defaultValue `'sfid,sfkbid,sfkavid,filetype'`
    */
-  @api fieldsToIncludeInCitations = 'sfid,sfkbid,sfkavid';
+  @api fieldsToIncludeInCitations = 'sfid,sfkbid,sfkavid,filetype';
```

Comment:
Lucille brought up a good point the other day,

If a customer decides to set different `fieldsToIncludeInCitations` here, they will essentially override our fields to include that are necessary for some features.

I think the default for `fieldsToIncludeInCitations` could become `=''` but we would always join with the defaults?

Reply:
sure I can do something like this to ensure that if user set it up wrong, it still sets the this.citationFields properly:
<img width="1149" height="251" alt="image" src="https://github.com/user-attachments/assets/58cddc5c-0042-4396-951f-9acff502437b" />

Reply:
where : `const DEFAULT_CITATION_FIELDS = ['sfid', 'sfkbid', 'sfkavid', 'filetype'];`